tools/apiview/parsers/csharp-api-parser/CSharpAPIParser/TreeToken/CodeFileBuilder.cs

```diff
@@ -231,72 +247,82 @@ private bool HasOnlyHiddenTypes(INamespaceSymbol namespaceSymbol)
             return namespaceSymbol.GetTypeMembers().All(t => IsHiddenFromIntellisense(t) || !IsAccessible(t));
         }
 
-        private void BuildType(List<APITreeNode> apiTree, INamedTypeSymbol namedType, bool inHiddenScope)
+        private void BuildType(List<ReviewLine> reviewLines, INamedTypeSymbol namedType, bool inHiddenScope)
         {
             if (!IsAccessible(namedType))
             {
                 return;
             }
 
-            bool isHidden = IsHiddenFromIntellisense(namedType);
-            var apiTreeNode = new APITreeNode();
-            apiTreeNode.Kind = APITreeNode.TYPE;
-            apiTreeNode.PropertiesObj.Add(APITreeNode.SUB_KIND, namedType.TypeKind.ToString().ToLowerInvariant());
-            apiTreeNode.Id = namedType.GetId();
-            apiTreeNode.Name = namedType.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat);
-
-            if (isHidden && !inHiddenScope)
+            bool isHidden = IsHiddenFromIntellisense(namedType) || inHiddenScope;
+            //Add empty line to separate types before current node begins
+            if (reviewLines.Count > 0)
             {
-                apiTreeNode.TagsObj.Add(APITreeNode.HIDDEN);
+                reviewLines.Add(new ReviewLine()
+                {
+                    IsHidden = isHidden
+                });
             }
 
-            BuildDocumentation(apiTreeNode.TopTokensObj, namedType);
-            BuildAttributes(apiTreeNode.TopTokensObj, namedType.GetAttributes());
-            BuildVisibility(apiTreeNode.TopTokensObj, namedType);
-            apiTreeNode.TopTokensObj.Add(StructuredToken.CreateSpaceToken());
+            var reviewLine = new ReviewLine()
+            {
+                LineId = namedType.GetId(),
+                Tokens = [],
+                IsHidden = isHidden
```

Comment:
Tokens is already initialized as an empty array.

```suggestion
                LineId = namedType.GetId(),
                IsHidden = isHidden
```